scss/bootscore_woocommerce/_wc_my_account_offcanvas.scss

```diff
@@ -2,25 +2,34 @@
 WooCommerce My Account Offcanvas
 --------------------------------------------------------------*/
 
-// Add show-password to non Woo pages
 #offcanvas-user {
-  .offcanvas-password.show-password-input {
-    position: absolute;
-    right: 0.7em;
-    top: 2rem;
-    cursor: pointer;
+  .woocommerce {
+
+    // form-login
+    .col2-set {
+
+      .col-1,
+      .col-2 {
+        width: 100%;
+      }
+    }
+
+    &-MyAccount-content {
+      p:last-child {
+        // Hide introtext, links do not link to my-account.php
+        display: none;
+      }
 
-    &::after {
-      font-family: WooCommerce;
-      speak: never;
-      font-weight: 400;
-      font-variant: normal;
-      text-transform: none;
-      line-height: 1;
-      -webkit-font-smoothing: antialiased;
-      margin-left: 0.618em;
-      content: "\e010";
-      text-decoration: none;
     }
   }
-}
\ No newline at end of file
+
+  // my-account
+  .col-md-4,
+  .col-md-8 {
+    width: 100%;
+  }
```

Comment:
Do we want to achieve this using filters instead? Just like we did with https://github.com/bootscore/bootscore/pull/577/commits/696e5024d7f776f269796e961ec23cfa2be457f6? Not really sure how we filter if it's the offcanvas, but I'm sure we can find a way. But I'll also be fine with this solution hahaha.

Reply:
The thing is that we have to readd the WC template to the file and I'm very happy that this file is removed.

Reply:
But the cols are set somewhere, right? We can add the filters in there.
But looking more closely, it being loaded from shortcode will complicate the way to find out where the filter is coming from. I think this is the easier method then.

Reply:
Ups, sorry. My bad. Here are the files with the cols:

- https://github.com/bootscore/bootscore/blob/include-my-account.php-in-user-offcanvas/woocommerce/myaccount/navigation.php
- https://github.com/bootscore/bootscore/blob/include-my-account.php-in-user-offcanvas/woocommerce/myaccount/my-account.php

Filter will be much more better

Reply:
I still haven't really thought of a way to implement the filter here, as I can't really do a check if the current location is the offcanvas. Will need to think about it more, or we do this solution for now.

Reply:
Think the CSS solution is fine, because the columns are in a Woo template and if they do an update to this file, I will try to find a way to remove them. But I'm sure Woo will do something completely new with blocks there in future. So I think we not should pay too much attention to that.

As mentioned, this PR will have a conflict with https://github.com/bootscore/bootscore/pull/647 which is much more urgent. I will fix this after https://github.com/bootscore/bootscore/pull/647 is merged.